src/services/bookstoreService.js

```diff
@@ -0,0 +1,24 @@
+import http from '../API-commons';
+
+import {
+  appsEndpoint, appIdEndpoint, itemIdEndpoint, booksEndpoint,
+} from '../modules/endpoints';
+
+const createApp = () => http.post(appsEndpoint);
+
+const createBook = (id) => http.post(appsEndpoint + appIdEndpoint(id) + booksEndpoint);
+
```

Comment:
- [ ] After creating the function to post to the API endpoint, you are to create an asynchronous action creator that returns a function and adds books through the API endpoint. The action creator will return a function that takes `dispatch` as a parameter. The function being returned will then dispatch the action `creator` addBook in your redux/books/books.js file. This asynchronous action creator should be dispatched in the `Form` component when a book is added. You can read this [article](https://www.bezkoder.com/react-redux-crud-example/) to have a clearer idea of what I am talking about.

#### Pay special attention to this part to have an idea of how to create the asynchronous action creator for creating a book using an API endpoint. 👍 

![image](https://user-images.githubusercontent.com/77106962/172716089-90d6e445-9411-49a4-ad2b-2b3f94de9390.png)